src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -599,7 +612,7 @@ def config_source_control(resource_group_name, name, repo_url, repository_type=N
     else:
         from azure.mgmt.web.models import SiteSourceControl, SourceControl
         if git_token:
-            sc = SourceControl(location, name='GitHub', token=git_token)
+            sc = SourceControl(location, token=git_token)
```

Comment:
This should be `source_control_name='GitHub'`, based on similar change you are making below

Reply:
Fixed